### PR TITLE
login_ctrl converted to es6

### PR DESCRIPTION
**Pull request by karinashenaz**

### CODE REVIEW

**Comment by Sjors** [comment]
There were some (important) comments in the original file (line 14-17, 47, etc). Can you move those over?

**Comment by karinashenaz** [comment]
Inserted the comments back

File: assets/js/controllers/login_ctrl.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        key: $scope.key,
+        email: email
+      };
+      $state.go("signup");
+    };
+    $cookieStore.remove('key');
+  // If BETA=1 is set in .env then in index.html/jade $rootScope.beta is set.
+  // The following checks are not ideal as they can be bypassed with some creative Javascript commands.
+    if ($rootScope.beta) {
+      return $http.post("/check_beta_key_unused", {
```
Comment (Sjors) [inline assets/js/controllers/login_ctrl.js]: return statement not needed

File: assets/js/controllers/login_ctrl.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+  $scope.login = () => {
+    if ($scope.busy) {
+      return;
+    }
+    $scope.busy = true;
+    Wallet.clearAlerts();
+    const error = (field, message) => {
+      $scope.busy = false;
+      if (field === "uid") {
+        return $scope.errors.uid = message;
```
Comment (Sjors) [inline assets/js/controllers/login_ctrl.js]: This (and possibly other) return statement is not needed.

**Pull request merged**